docs/guides/primitives.mdx

```diff
@@ -81,32 +80,53 @@ sampler.run([
 ])
 ```
 
-## How to use Qiskit primitives
+## Benefits of Qiskit primitives
+
+With primitives, Qiskit users can write quantum code for a specific QPU without having to explicitly
+manage every detail. Also, because of the additional layer of abstraction, you might be able to more easily
+access advanced hardware capabilities of a given provider. For example, with Qiskit Runtime primitives,
+you can take advantage of the latest advancements in error mitigation and suppression by toggling options such as the primitive's [`resilience_level`,](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.EstimatorOptions#resilience_level) rather than building your own implementation of these techniques.
+
+For hardware providers, implementing primitives natively means you can provide your users with a more “out-of-the-box”
+way to access your hardware features (such as advanced post-processing techniques). It is therefore easier for your users to benefit from your hardware's
+best capabilities.
+
+## Definition and implementation of Qiskit primitives
+
+The Qiskit primitives are defined by open-source primitive base classes, from
+which different providers can derive their own Sampler and Estimator implementations. Among the implementations
+that use Qiskit, you can find reference primitive implementations for local simulation in the [`qiskit.primitives`](/api/qiskit/primitives) module.
+Providers like [Qiskit Runtime](/api/qiskit-ibm-runtime/runtime_service) enable access to appropriate QPUs through native implementations of
+their own primitives.
+
+
+The [`qiskit.primitives`](/api/qiskit/primitives) module enables the development of primitive-style quantum programs and was specifically
+designed to simplify switching between different types of quantum compute resources. As described previously, there are two types of Qiskit primitives: the base class, and their implementations.
```

Comment:
it looks like theres some funny indenting here? 
<img width="872" alt="Screenshot 2024-10-08 at 1 03 53 PM" src="https://github.com/user-attachments/assets/11f37b2b-eddc-4bc1-9f70-3210e62d293e">